objects/LuaScript.ttslua

```diff
@@ -300,6 +300,15 @@ expansionDecks = {
   sites2 = '486402',
   guardians = 'b876d5'
 }
+missingExpeditionDecks = {
+  --artifacts = 'f61f69',
+  --items = '5076de',
+  assistants = 'b640aa',
+  --idols = '9988c4',
+  --sites1 = '37a6dd',
+  --sites2 = '486402',
+  --guardians = 'b876d5'
```

Comment:
Are the entries intentionally commented out?

Reply:
Yeah they're place holders for as we implement the rest of the decks/items